$NetBSD: patch-am,v 1.4 2012/03/08 10:11:24 he Exp $

This test is timing-dependent, allow more time for the asynchronous
callbacks to complete and/or a slower build + test machine.
(Reportedly fixed in newer versions.)

--- t/pmc/nci.t.orig	2012-03-06 08:42:14.000000000 +0000
+++ t/pmc/nci.t
@@ -1917,7 +1917,7 @@ OUTPUT
 
     # Need to force reschedule to see async callbacks.
     # Chandon TODO: Is this a bug?
-    sleep 0.001
+    sleep 0.02
 
     # reset int_cb_D4 to 1
     int_cb_D4[0] = 1
